src/Resources/laravel-debugbar.css

```diff
@@ -352,14 +375,25 @@ ul.phpdebugbar-widgets-timeline table.phpdebugbar-widgets-params {
     font-size: 11px;
 }
 
+ul.phpdebugbar-widgets-timeline table.phpdebugbar-widgets-params td {
+    border: none !important;
+}
+
+ul.phpdebugbar-widgets-timeline table.phpdebugbar-widgets-params td:first-child {
+    padding: 0 10px;
+}
+
 div.phpdebugbar-widgets-messages div.phpdebugbar-widgets-toolbar {
-    width: calc(100% - 20px);
```

Comment:
Normally these `-20 px` were added when there is a lot of data the scrollbar appears, then a presentation problem occurred in the toolbar, maybe is a regression

Reply:
Perhaps many of these changes should be applied first to [maximebf/php-debugbar](https://github.com/maximebf/php-debugbar)

Reply:
Do you know perhaps of a specific tab that caused scrolling issues? And we're talking about an unwanted horizontal scrollbar right? I did get a horizontal scrollbar with the change you marked but I applied more changes to get rid of it.

I can certainly extract changes to the base package. I was just so far in with the changes that I thought it might make more sense to first gather feedback and then extract (if people would agree with it).